### PR TITLE
Update measure tools to use eslint and raise itwin.js minimum

### PR DESCRIPTION
**Pull request by calebmshafer**

### CODE REVIEW

File: packages/measure-tools/src/api/MeasureToolsLoggerCategory.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ export enum MeasureToolsLoggerCategory {
   Root = "Measure-Tools",
 }
 
+// eslint-disable-next-line @typescript-eslint/no-redeclare
```
Comment (aruniverse) [inline packages/measure-tools/src/api/MeasureToolsLoggerCategory.ts:14]: where is this being redeclared?

**Review by aruniverse — APPROVED** [review]

**Comment by aruniverse** [comment]
did you bump the core ver? or mean to?

File: packages/measure-tools/src/api/MeasureToolsLoggerCategory.ts (hunk 1 above)
Comment (calebmshafer) [reply packages/measure-tools/src/api/MeasureToolsLoggerCategory.ts:14]: That's due to the namespace and enum. I really don't understand why both exist but thought that'd be a different PR, and later problem to fix.

**Comment by calebmshafer** [comment]
> did you bump the core ver? or mean to?

@aruniverse yup, just pushed that change. A couple of the changes I made to clean up the lint were added in 2.19.

**Comment by calebmshafer** [comment]
@aruniverse mind taking another quick look? Pushed a few more updates while I was already updating the minimum version needed.

Also, do we want to try and get a release out with the cherry-picked changes and the 2.17 minimum prior to updating all the way to 2.19?

**Comment by aruniverse** [comment]
> @aruniverse mind taking another quick look? Pushed a few more updates while I was already updating the minimum version needed.
> 
> Also, do we want to try and get a release out with the cherry-picked changes and the 2.17 minimum prior to updating all the way to 2.19?

new changes look fine to me, i dont think theres much usage of this pkg, so id go straight to 2.19, but whatever is easiest for you

**Comment by calebmshafer** [comment]
@aruniverse that's easier for me. We'll go straight there. Thanks!

**Pull request merged**